DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: x13binary
 Type: Package
 Title: Provide the 'x13ashtml' Seasonal Adjustment Binary
-Version: 0.1.0
-Date: 2016-01-18
+Version: 0.1.1
```

Comment:
I suggest to mark this as 0.1.0.1, then 0.1.0.2, ... to reserve 0.1.1 for when we are ready to upload.

Or do you think this should go as is?